package.json

```diff
@@ -201,7 +201,9 @@
     "micromatch": "^4.0.8",
     "express": "^4.21.0",
     "path-to-regexp@^1.7.0": "^1.9.0",
-    "path-to-regexp@^6.2.0": "^6.3.0"
+    "path-to-regexp@^6.2.0": "^6.3.0",
+    "rollup@^2.43.1": "^2.79.2",
```

Comment:
@nicholas-codecov we can do this upgrade without touching package.json (like `npm upgrade` used to do) with these commands:

```yarn add rollup@4.24```
```yarn dedupe rollup```
```yarn remove rollup```

Reply:
Seems to me like resolutions is more for overriding the ranges when needed, but it seems there's no range we need to override here.

Reply:
Can you write up doc on this in Gazebo Guidelines please @spalmurray-codecov 🙏

Reply:
Sure thing